src/bs1770demo/README.md

```diff
@@ -0,0 +1,6 @@
+Implementation of BS.1770-4 as defined in Recommendation ITU-R BS.1770-4
+See LICENSE.md for terms.

```

Comment:
LICENSE.md is not part of this commit.

Reply:
Thanks Ludo!

I have removed the references to the non-existent LICENSE.md in both README-files.